test/unit/CoinbaseOracle.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.17;
+
+import {CommonTest} from "../Common.sol";
+
+import {BaseController} from "../../src/controllers/BaseController.sol";
+import {CoinbaseOracleSourceAdapter} from "../../src/adapters/source-adapters/CoinbaseOracleSourceAdapter.sol";
+import {DecimalLib} from "../../src/adapters/lib/DecimalLib.sol";
+import {IAggregatorV3Source} from "../../src/interfaces/chainlink/IAggregatorV3Source.sol";
+import {CoinbaseOracle} from "../../src/oracles/CoinbaseOracle.sol";
+
+contract CoinbaseSourceAdapterTest is CommonTest {
+    CoinbaseOracle coinbaseOracle;
```

Comment:
There is also some inconsistency in naming here. We call it `CoinbaseOracle` but the fine is `CoinbaseSourceAdapterTest` but the the file within `source-adapters` is `CoinbaseOracleSouceAdapter`. I think we drop `Oracle` from all of these like mentioned here https://github.com/UMAprotocol/oval-contracts/pull/6#discussion_r1602912004